dts/arm/sifli/sf32lb52x-ram012.dtsi

```diff
@@ -13,7 +14,21 @@
 	soc {
 		ram012: memory@20000000 {
 			compatible = "mmio-sram";
-			reg = <0x20000000 DT_SIZE_K(512)>;
+			reg = <0x20000000 DT_SIZE_K(511)>;
+		};
+		sram0_shared: memory@2007fc00 {
+			compatible = "zephyr,memory-region", "mmio-sram";
+			reg = <0x2007fc00 DT_SIZE_K(1)>;
+			device_type = "memory";
+			zephyr,memory-region = "sram0_shared";
+			zephyr,memory-attr = <(DT_MEM_ARM(ATTR_MPU_RAM_NOCACHE))>;
+			};
```

Comment:
Incorrect indentation. Also, please add empty lines between the DT nodes. You can find more info on the expected style here: https://docs.zephyrproject.org/latest/contribute/style/devicetree.html